### PR TITLE
ci(core): add PR title linting to GH ci

### PR DESCRIPTION
**Pull request by Archento**

### CODE REVIEW

File: .github/workflows/pr-title-linting.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            chore
+            docs
+            feat
+            fix
+            refactor
+            style
+            perf
+            ci
+            test
+            revert
```
Comment (jrriehl) [inline .github/workflows/pr-title-linting.yml:27]: Just a couple more suggestions:
```suggestion
            revert
            integration
            misc
```

Comment (jrriehl) [reply .github/workflows/pr-title-linting.yml:27]: Oh I see you made it a scope.

Comment (Archento) [reply .github/workflows/pr-title-linting.yml:27]: yes, I think that makes sense in the current structure :)

**Review by jrriehl — APPROVED** [review]
LGTM

**Pull request merged**